WeakAuras/Prototypes.lua

```diff
@@ -2549,7 +2549,7 @@ Private.event_prototypes = {
     init = function(trigger)
       local ret = [=[
         local useWatched = %s
-        local factionID = useWatched and select(6, GetWatchedFactionInfo()) or %q
+        local factionID = useWatched and C_Reputation.GetWatchedFactionData().factionID or %q
```

Comment:
unfortunately it's not this easy, since the various flavors of classic do not yet have this API change yet. Standard practice is to just add a shim which checks the client flavor, like so: https://github.com/WeakAuras/WeakAuras2/commit/9b214ef218531993d1fb4d32343e23c95d9d58e6

Reply:
Should be fine now, I think? Thank you for the help